docs/mesos/Dockerized27Deployment.md

```diff
@@ -0,0 +1,226 @@
+
+      sudo mkdir /keys
+      sudo openssl genrsa -f4  -out /keys/key.pem 4096
+      sudo openssl req -new -batch -x509  -days 365 -key /keys/key.pem -out /keys/cert.pem
+
+# Prepare Master
+Perform the following steps on one host which will be designated the "Master".
+
+## 1. Configure your firewall
+
+You will either need to configure the firewalls on each node in your cluster (recommended) to allow access to the cluster services or disable it completely.  Included in this section is configuration examples for `firewalld`.
+
+Master node(s) require
+
+| Service Name | Port/protocol     |
+|--------------|-------------------|
+| zookeeper    | 2181/tcp          |
+| mesos-master | 5050/tcp          |
+| etcd         | 2379/tcp 4001/tcp |
+| marathon     | 8080/tcp          |
+
+Example `firewalld` config
+
+    sudo firewall-cmd --zone=public --add-port=2181/tcp --permanent
+    sudo firewall-cmd --zone=public --add-port=5050/tcp --permanent
+    sudo firewall-cmd --zone=public --add-port=2379/tcp --permanent
+    sudo firewall-cmd --zone=public --add-port=4001/tcp --permanent
+    sudo firewall-cmd --zone=public --add-port=8080/tcp --permanent
+    sudo systemctl restart firewalld
+    
+# 2. Zookeeper
+Download the Zookeeper image, as well as the systemd service which will ensure Zookeeper is kept running:
+
+    sudo docker pull jplock/zookeeper:3.4.5
+    wget https://raw.githubusercontent.com/projectcalico/calico-mesos/0.27/dockerized-mesos/config/units/zookeeper.service
+    sudo cp zookeeper.service /usr/lib/systemd/system/
+    sudo systemctl enable zookeeper.service
+    sudo systemctl start zookeeper.service
+
+## 3. Mesos Master
+Before running the Mesos-Master process, we'll set the IP address of the Master to connect to the Mesos cluster.  Run the following command, replacing `<MASTER_IP>` with the Master's IP address.
+
+    sudo sh -c 'echo IP=<MASTER_IP> > /etc/sysconfig/mesos-master'
+
+Then create and enable the `mesos-master` unit, which starts a Docker container running Mesos-Master:
+
+    sudo docker pull calico/mesos-calico:0.27
+    wget https://raw.githubusercontent.com/projectcalico/calico-mesos/0.27/dockerized-mesos/config/units/mesos-master.service
+    sudo cp mesos-master.service /usr/lib/systemd/system/
+    sudo systemctl enable mesos-master.service
+    sudo systemctl start mesos-master.service
+
+## 4. Etcd
+
+`etcd` needs your fully qualified domain name to start correctly.  The included
+unit file looks for this value in `/etc/sysconfig/etcd`.
+
+    sudo sh -c 'echo FQDN=`hostname -f` > /etc/sysconfig/etcd'
+    sudo docker pull quay.io/coreos/etcd:v2.2.0
+    wget https://raw.githubusercontent.com/projectcalico/calico-mesos/0.27/dockerized-mesos/config/units/etcd.service
```

Comment:
Don't use wget.  curl is installed by default but wget is not.... and we're using both wget and curl